commands/plugin_marketplace_e2e_test.go

```diff
@@ -141,3 +141,27 @@ func removePluginIfInstalled(s *MmctlE2ETestSuite, pluginID string) {
 		s.Require().Contains(appErr.Error(), "Plugin is not installed.")
 	}
 }
+
+func (s *MmctlE2ETestSuite) TestPluginMarketplaceListCmd() {
+	s.SetupTestHelper().InitBasic()
+
+	s.RunForSystemAdminAndLocal("List Marketplace Plugins for Admin User", func(c client.Client) {
+		printer.Clean()
+
+		err := pluginMarketplaceListCmdF(c, &cobra.Command{}, nil)
+
+		s.Require().Nil(err)
+		s.Require().Len(printer.GetErrorLines(), 0)
+	})
```

Comment:
Can we add a way to check that there is some output from the command's execution? Although it would be brittle to check for a specific plugin as part of it, I think we can safely assume that there will always be some output from the list

Reply:
@mgdelacroix  sure will include that